packages/manager/modules/cloud-connect/src/details/service-keys/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,15 @@
+{
+  "cloud_connect_service_keys_status_doing": "Faire",
```

Comment:
Issue raised from translation team:

```suggestion
  "cloud_connect_service_keys_status_doing": "À faire",
```

Reply:
A translation request has already been created, will get this updated. Thanks